build.gradle

```diff
@@ -103,12 +84,13 @@ task checkVersion {
 		try {
 			def xml = new URL("https://maven.quiltmc.org/repository/release/org/quiltmc/qsl/maven-metadata.xml").text
 			def metadata = new groovy.xml.XmlSlurper().parseText(xml)
-			def versions = metadata.versioning.versions.version*.text();
+			def versions = metadata.versioning.versions.version*.text()
+
 			if (versions.contains(version)) {
 				throw new RuntimeException("${version} has already been released!")
 			}
-		} catch (FileNotFoundException ex) {
-			// Ignored
+		} catch (FileNotFoundException ignored) {
+			// No existing version of library found
```

Comment:
Maybe it may be better to print something, just in case someone later on ends up moving a library and Gradle can't find it.